### DISCUSSION TITLE
Having issues getting 12 step meeting list plugin output to display properly using the Avada theme on mobile phones

### DISCUSSION THREAD

**Discussion by JoeBryan7873** [code4recovery/12-step-meeting-list#1115] [Q&A]
HI,

I'm building a new website for an AA group on a staging server using the Avada Wordpress theme. The 12 step meeting list looks fine on the desktop but isn't displaying properly when viewed on mobile devices. Please see attached screenshot
![Plugin not displaying properly on mobile devices on  Avada staging server]

I'm using the Legacy UI setting as when I try the TSML UI the meeting list doesn't show up. Not sure why.

(https://user-images.githubusercontent.com/130705835/231879863-a70ac322-f55f-460a-b75e-a01046740e84.jpg)

I'm not sure how to fix this and any help would be greatly appreciated. The client really wants to use the Avada theme and I'm guessing something about the theme styles is causing this issue.  I've tried everything I can think of with no success. I have ticket out with the Theme developer but thought I would try from this end as well.

Thank you in advance for the help

**Comment by joshreisner** [comment]
hm, looks like it needs a CSS tweak to remove padding (i'm guessing) the theme is adding to the page on mobile. we can suggest a CSS fix if you can give us a link!

you might also try the "TSML UI" appearance - this has some advantages over the legacy appearance, although i doubt switching by itself would fix the padding issue

**Reply by JoeBryan7873** [reply]
Thank you, I really appreciate it. I figured it had something to do with CSS

Here is the link to the Meeting list on the staging server
https://dev1.123marketingpros.com/meetings/

I did try switching to the TSML UI but for some reason when I switch from the Legacy UI none of the meetings show up in the list, not sure why.  Also I think the client group wants to keep the same interface if possible. 

Really appreciate your help!

**Reply by joshreisner** [reply]
hm strange! if you ever want to debug the TSML UI thing you can check the javascript console, or send us a link and we can take a look

here's some CSS you should be able to drop in to fix the margins:

```
#tsml {
  margin: -30px -30px 0;
  width: calc(100% + 60px);
}
```

**Reply by JoeBryan7873** [reply]
That did it! Thank you so much, I can't tell you how much I appreciate it! 
I'll get back to you on the TSML UI issue if we decide to change
